doc/cephadm/upgrade.rst

```diff
@@ -48,6 +48,27 @@ The automated upgrade process follows Ceph best practices.  For example:
 Starting the upgrade
 ====================
 
+.. note::
+
+   Cephadm by default reduces `max_mds` to `1`. This can be disruptive for large
+   scale CephFS deployments because the cluster cannot easily reduce active MDS(s)
+   to `1`. Therefore, To upgrade MDS(s) without reducing `max_mds`, the
+   `fail_fs` option can to be set to `true` using `ceph orch` (default value is
+   `false`):
+
+   .. prompt:: bash #
+
+      ceph config set mgr mgr/orchestrator/fail_fs true
+
+   This would:
```

Comment:
The `config set` doesn't perform the steps below per se, isn't it? I think that should be made clear (that the upgrade step via cephadm would do this).

Reply:
I'll update it and it clearing mentioning it's a step that needs to be done before initiating the upgrade

Reply:
thx